.github/workflows/release-server-to-docker.yml

```diff
@@ -9,18 +9,27 @@ on:
         description: 'Version'     
         required: true
 
+env:
+  IRONBANK_HEIMDALL_PROJECT_ID: 5450 # this is for heimdall (non mainline) - I think these can be in-line envs instead of supplied by github repo/org level secrets/values since each push/release workflow will have a unique ironbank id due to the mainline vs release + heimdalllite vs heimdall matrix
+
+# pull request sha: ${{ github.event.pull_request.head.sha }} - will only be used during testing | on push sha: ${{ github.sha }} - will likely be used with mainline | new tag: ${{ steps.format-tag.outputs.replaced }} or underscored version: ${{ env.VERSION_UNDERSCORED }} - will be used during releases
+
 jobs:
   docker:
     runs-on: ubuntu-22.04
     steps:
-      - name: Run string replace # remove the v from the version number before using it in the docker tag
+      - name: Run string replace to remove the v from the version number before using it in the docker tag
         uses: frabert/replace-string-action@v2
         id: format-tag
         with:
           pattern: 'v'
           string: '${{ github.event.release.tag_name || github.event.inputs.version}}'
           replace-with: ''
           flags: 'g'
+      - name: Convert periods to underscores in version string
```

Comment:
wrote underscores, but meant hyphens.  need to change the var name and comment

Reply:
https://github.com/mitre/heimdall2/pull/6448/commits/0e0e0baa5a50fd394c36c234fc2b87ae56985a45